### PR TITLE
feat: teams ui

### PR DESCRIPTION
**Pull request by Surya-n**

### CODE REVIEW

**Comment by DarkPhoenix2704** [comment]
@Surya-n If it is UI change, include screenshots

**Comment by Surya-n** [comment]
Screenshot 2023-09-18 223803

**Comment by DarkPhoenix2704** [comment]
Good job

File: apps/web/pages/org/[id]/teams.tsx [hunk 1]
```diff
@@ -1,8 +1,49 @@
...
+          </div>
+
+        </div>
+        
+  <input
+    type="search"
+    className="block w-sm text-[16px] py-2 px-3 rounded-lg border border-gray-300 shadow-sm focus:ring focus:ring-brand-pink-100  focus:border-brand-blue-400 mt-4"
+    placeholder="Find team or person"
+  />
+    <div className="flex flex-wrap flex-col gap-y-[25px] gap-x-[75px] justify-center items-center p-4  md:justify-start lg:flex-row lg:items-center mr-6 ml-6 ">
```
Comment (sreehari2003) [inline apps/web/pages/org/[id]/teams.tsx:37]: use radix or shadcb ui input dont use default input

File: apps/web/pages/org/[id]/teams.tsx [hunk 2]
```diff
@@ -1,8 +1,49 @@
...
+            <h1 className="text-[30px] text-center font-bold">23</h1>
+  <p className="text-[12px] font-bold text-center">PROS</p>
+            </div>
+          </div>
+
+        </div>
+        
+  <input
+    type="search"
+    className="block w-sm text-[16px] py-2 px-3 rounded-lg border border-gray-300 shadow-sm focus:ring focus:ring-brand-pink-100  focus:border-brand-blue-400 mt-4"
```
Comment (sreehari2003) [inline apps/web/pages/org/[id]/teams.tsx:34]: please run the linter

File: apps/web/theme/style.css [hunk 3]
```diff
@@ -75,4 +75,4 @@
   body {
     @apply bg-background text-foreground;
   }
-}
```
Comment (sreehari2003) [inline apps/web/theme/style.css:78]: what is this ?

File: apps/web/views/dashboard/components/Teams.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      </div>
+      
+      <div className="flex items-center space-x-2 ">
+        <img src="https://img.freepik.com/free-photo/beautiful-woman-street_23-2147654273.jpg?w=2000" alt="Member 4" className="w-8 h-8 rounded-full" />
+        <div>
+        <div className="font-semibold text-md">Surya</div>
+          <div className="text-xs text-muted-foreground">surya@email.com</div>
+        </div>
+      </div>
+    </CardContent>
```
Comment (sreehari2003) [inline apps/web/views/dashboard/components/Teams.tsx:46]: there should be spacing b/w easch user give a gap

File: apps/web/pages/org/[id]/teams.tsx [hunk 5]
```diff
@@ -1,8 +1,49 @@
...
     return (
-        <div>teams</div>
+        <div className=" p-6">
+        <h1 className="text-xs">FOSSHACK 3.0</h1>
+        <div className="flex justify-between items-center ">
+          <h1 className="text-[38px] font-bold">Registrations</h1>
+          
+          <div className="flex">
+            <div className="items-center w-[auto] h-[auto] bg-brand-pink-100 rounded-lg p-2 mr-4">
+              <h1 className="text-[30px] text-center font-bold">124</h1>
```
Comment (sreehari2003) [inline apps/web/pages/org/[id]/teams.tsx:13]: 4px = 1 in tailwind border-3 border-[red]


so 38 px should be 9.5 ie make text-9.5

**Comment by Surya-n** [comment]
Ss

File: apps/web/views/dashboard/components/Teams.tsx [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+                            <div className="text-xs text-muted-foreground">surya@email.com</div>
+                        </div>
+                    </div>
+
+                    <div className="flex items-center space-x-2">
+                        <img
+                            src="https://img.freepik.com/free-photo/beautiful-woman-street_23-2147654273.jpg?w=2000"
+                            alt="Member 2"
+                            className="w-8 h-8 rounded-full"
+                        />
```
Comment (DarkPhoenix2704) [inline apps/web/views/dashboard/components/Teams.tsx:31]: Add `object-cover` class to image

**Review by DarkPhoenix2704 — CHANGES_REQUESTED** [review]

**Pull request merged**